play/PlayerRules/Basic Rules.txt

```diff
@@ -250,17 +279,17 @@ Species. See chapter 1: Races if you haven't yet.
 
 Leveling up:
 Gain feats according to the table below.
-With Charisma 9 or higher, receive more feats. The feats listed in the table are totals, 
+With Intelligence 9 or higher, receive more feats. The feats listed in the table are totals, 

```

Comment:
Because of secondary skills putting allowing skill point gain to happen with Charisma and Intelligence, intelligence can easily become a dump stat if you don't need nanites. Moving the charisma feat bonus to intelligence and giving charisma a unique feature, seemed like an appropriate balance